lib/travis/build/git/clone.rb

```diff
@@ -88,14 +88,19 @@ def delete_netrc
           end
 
           def github?
+            source_host_name.downcase == 'github.com' || source_host_name.downcase.end_with?('.github.com')
```

Comment:
could this just be `source_host_name.downcase.end_with?('github.com')`? or `source_host_name.downcase =~ /\bgithub\.com/`